src/utils/upload-file-browser.js

```diff
@@ -4,10 +4,20 @@ const { jsonApiError } = require('./simple-fetch')
 const { isFunction } = require('./typecheck')
 
 const uploadFileBrowser = (options = {}) => {
+  const metadataHeader = 'x-amz-meta-'
+  const versions = options.versions
+
   return new Promise((resolve, reject) => {
     const xhr = new window.XMLHttpRequest()
     xhr.open('PUT', options.uploadUrl)
     xhr.setRequestHeader('Content-Type', options.fileMimeType || '')
+
+    if (typeof versions !== 'undefined') {
```

Comment:
Just use `isEmpty(versions)`. If we don't have this package yet, just install it.

It really comes in handy. https://www.npmjs.com/package/is-empty